scripts/Tools/code_checker

```diff
@@ -85,12 +91,6 @@ def check_code(dir_to_check, num_procs, files):
 
     Returns True if all files had no problems
     """
-    # Pylint won't work right if the imports within the checked files fails
```

Comment:
Because I provide the python path in scripts_regression_tests and the path should be to 
scripts/Tools and utils/python (and not to .) to reproduce the behavior we expect outside of pylint.